nautobot_circuit_maintenance/models.py

```diff
@@ -14,6 +15,9 @@
 
 from .choices import CircuitImpactChoices, CircuitMaintenanceStatusChoices, NoteLevelChoices
 
+PLUGIN_SETTINGS = settings.PLUGINS_CONFIG.get("nautobot_circuit_maintenance", {})
+MAX_RAW_NOTIFICATION_SIZE = 1000
```

Comment:
Misleading name? We can configure a higher value than this if desired, right? Perhaps `DEFAULT_RAW_NOTIFICATION_SIZE` would be better?

This only applies to the RawNotification, right? The CircuitMaintenance will still store the full text of the email?

Reply:
The `CircuitMaintenance` doesn't store the email, only the `RawNotification`, so here we are actually truncating the original notification in the DB, but I believe that we should add some limit to prevent against big email attachements we could receive.
Maybe we could define a lower level to get some minimum size?